.gitignore

```
@@ -10,6 +10,7 @@
 
 # production
 /build
+/dist
```

Maybe it will be better in the case of creating one more parent folder with, for example, the backend. Also, you need to remove the slash with `node_modules` for the same reason.
```suggestion
build
dist
```